packages/gestalt/src/Image.tsx

```diff
@@ -226,7 +226,7 @@ export default class Image extends PureComponent<Props> {
           onLoad={this.handleLoad}
           role={role === 'presentation' ? 'presentation' : undefined}
           sizes={sizes}
-          src={src}
+          src={typeof src === 'string' && src !== '' ? src : undefined}
```

Comment:
can we do 

`          src={typeof src === 'string' && src === '' ?  undefined : src}
`

Reply:
I just want to make sure src={undefined} also makes it